packages/backend/src/models/CatalogModel/CatalogModel.ts

```diff
@@ -305,6 +313,13 @@ export class CatalogModel {
                 );
             });
 
+        if (context === CatalogSearchContext.SPOTLIGHT) {
+            catalogItemsQuery = catalogItemsQuery.where(
```

Comment:
I can see a filer here, we should add an index for this column

Reply:
TODO: set this column to an index

Reply:
done here: https://github.com/lightdash/lightdash/pull/13381/commits/1dbdbde9b4e4f416cdcdaeea2726f79ae43a15b5